tutordiscovery/templates/discovery/apps/settings/partials/common.py

```diff
@@ -55,7 +55,7 @@
 
 LOGGING["handlers"]["local"] = {
     "class": "logging.handlers.WatchedFileHandler",
-    "filename": "/var/log/discovery.log",
+    "filename": "/openedx/data/logs/discovery.log",
```

Comment:
I realize that my comment was not very clear: I meant to remove the "local" handler from the list of available handlers, as well from all loggers. That's because the "local" handler is created in the upstream course_discovery repo, and added to all loggers by default. With your change, I cannot build the discovery Docker image anymore:

```
Traceback (most recent call last):
  File "/usr/lib/python3.8/logging/handlers.py", line 855, in _connect_unixsocket
    self.socket.connect(address)
FileNotFoundError: [Errno 2] No such file or directory
```

It's ok, I'll merge your PR and push the change directly to the maple branch.